src/person/simple_serializers.py

```diff
@@ -19,6 +19,16 @@ class SimpleUserSerializer(serializers.ModelSerializer):
     interests = InterestSerializer(source="person.interests", many=True)
     groups = GroupSerializer(source="person.groups", many=True)
     prompts = serializers.SerializerMethodField("get_prompts")
```

Comment:
your entire serializer is just querying everything about a `Person`... is there a reason why you've not made a `PersonView` instead and queried the person directly? this is extremely inefficient, and serializers can slow down your code a lot, bc there is no DB optimization here

Reply:
hmmm i don't really understand how the db optimization works if the fields have to be fetched regardless, maybe you can walk me through it on Wednesday?